bioframe/ops.py

```diff
@@ -1382,14 +1388,15 @@ def trim(
         }
         inferred_view = True
 
+    ckv, skv, ekv = _get_default_colnames() if cols_view is None else cols_view
     view_df = construction.make_viewframe(
-        view_df, view_name_col=view_name_col, cols=cols
-    )
+        view_df, view_name_col=view_name_col, cols=[ckv, skv, ekv]
+    ).rename(columns=dict(zip([ckv, skv, ekv], [ck, sk, ek])))
 
     if inferred_view:
-        print('correct')
+        print("correct")
```

Comment:
can we delete this print?

Reply:
also can line 1423 be moved earlier, with the other inferred view stuff?

```python
if inferred_view:
        view_name_col = ck```

Reply:
otherwise looks good-- thanks @Phlya !

Reply:
Thank you @gfudenberg !